rfcs/template.md

```diff
@@ -0,0 +1,120 @@
+# RFC Title
+
+authors:
+
+- `[Author1](Github Link for Author)`
+
+necessary reviewers:
+
+- [ ] Reviewer 1
+- [ ] Reviewer 2
+- [ ] Reviewer 3
```

Comment:
Does this differ from the "request review" feature and/or "review required by codeowners" feature in GH?

Reply:
Yeah. Good question. It's not that codeowners are always the stakeholders per see, but if we use the PR request review correctly, we can remove this. I'll make this a note instead.